<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.xxZULRR' shared service.

return $this->privates['.service_locator.xxZULRR'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('atendimento' => function (): \Novosga\Entity\Atendimento {
    return ($this->privates['.errored..service_locator.xxZULRR.Novosga\Entity\Atendimento'] ?? $this->load('getAtendimento4Service.php'));
}, 'service' => function (): \App\Service\TicketService {
    return ($this->services['App\Service\TicketService'] ?? $this->load('getTicketServiceService.php'));
}, 'translator' => function () {
    return ($this->services['translator'] ?? $this->getTranslatorService());
}));
